.changeset/yellow-chairs-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+Added `CodeBlock` component
```

Comment:
```suggestion
`CodeBlock` - Added new component
```

Reply:
Updated in 1294af8a31f6e3b4035a8ae3c01a2f6b5acdd790